docs/SalesforceBatchSource-batchsource.md

```diff
@@ -0,0 +1,36 @@
+# Salesforce Batch Source
+
+
+Description
+-----------
+This source reads sObjects from Salesforce.
+Examples of sObjects are opportunities, contacts, accounts, leads, any custom object, etc.
+
+The data which should be read is specified using SOQL queries (Salesforce Object Query Language queries)
+
+Configuration
+-------------
+
+**Reference Name:** Name used to uniquely identify this source for lineage, annotating metadata, etc.
+
+**Username:** Salesforce username.
+
+**Password:** Saleforce password.
+
+**Client Id:** Application Client Id. This is also called "Consumer Key" in Salesforce UI.
```

Comment:
Can we call this consumer key - why say something different than what it is in SF.

Reply:
Well in most docs etc. it's called clientId, so I think it makes more sense to call it than. It's just one place when you set it for application for some reason it's "Consumer key". Not sure why.

Reply:
Example:
https://help.salesforce.com/articleView?id=000205876&language=en_US&type=1